### PR TITLE
Update application icon and description

### LINKED ISSUE #117: Give this thing a proper name and logo
Let's start the discussion, do we really want to be calling this MLVET for the final product? I feel like we can come up with something nicer :)

Also let's get thinking about a logo (maybe) and app icon (definitely)

### PR DESCRIPTION
**Pull request by Irmy**
## Summary

Update packaging details of application<hr/>

### Why is this change needed?
So our application doesn't look like electron
<hr/>

### What did you change?
Update packaging details such as MLVET logo in application rather than electron logo, update application description rather than electron boilerplate description and version number. 
Version number was chosen as 0.2.5 since application has not been released, but now on second MVP and week 5 ( because why not). 
<hr/>

### Testing details if applicable
Please check how the packaging works on MAC OS and Linux OS
<hr/>

### Any sections in particular you want reviewed/discussed
Please feel free to advise on another version number if current one is not suitable. 
<hr/>

### OS

- [ ] Linux
- [ ] MacOS
- [x] Windows

### CODE REVIEW

File: release/app/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "electron-react-boilerplate",
```
Comment (chloebrett) [inline release/app/package.json]: Update this too

**Review by chloebrett — APPROVED** [review]
Looks good

**Review by MutazAshhab — APPROVED** [review]

**Comment by MutazAshhab** [comment]
Looks amazing!

image

**Review by exquisitebin — APPROVED** [review]
image

I think this is due to the "name" element that Chloe addressed.
Otherwise it's looking fantastic!

**Pull request merged**